codespell.py

```diff
@@ -34,33 +49,17 @@ def splitCamelCase(word):
     return result
 
 
-# Map file suffix to file type.
 def getMimeType(filepath):
-
-    suffix2mime = { '.h': 'text/x-c++',
-                    '.cxx': 'text/x-c++',
-                    '.c': 'text/x-c++',
-                    '.hxx': 'text/x-c++',
-                    '.py': 'text/x-python',
-                    '.ruby': 'text/x-ruby',
-                    '.java': 'text/x-java-source',
-                    '.txt': 'text/plain',
-                    '.rst': 'text/plain',
-                    '.md': 'text/plain',
-                  }
+    """Map `filepath`` extension to file type."""
     name, ext = os.path.splitext(filepath)
+    return SUFFIX2MIME.get(ext, 'text/plain')
 
-    if ext in suffix2mime:
-        return suffix2mime[ext]
-    else:
-        return 'text/plain'
 
-
-#
-#  For a regular text file, we don't need to parse it for comments.  We
-#  just pass every line to the spell checked
-#
 def load_text_file(filename):
+    """
+    For a regular text file, we don't need to parse it for comments. We
```

Comment:
Docstrings should start right after the triple quotes.

Reply:
Could utilize [`pydocstyle`](https://pypi.org/project/pydocstyle/) to provide CI feedback about doc string styling. Whether the doc string is on the same line as the triple quotes or not depends on the situation that `pydocstyle` can enforce.

Reply:
@jamesobutler `pydocstyle` says it follows PEP 257 https://peps.python.org/pep-0257/, which does not seem to encourage having blank likes at the beginning of the docstrings. The `numpy` doc style does not, and `google` doc style does not either. At least by default, or across all their examples.

Reply:
`pydocstyle` recommends blank lines with the triple quotes depending on the situation. It recommends the following:
```python
def myfunction(self):
    """
    Summarize the function in the first line.

    Add more details about the the purpose and reasoning to provide more context in additional lines for the doc string.
    """
```
```python
def myfunction(self):
    """Summarize the function in the first line."""
```

Reply:
Then that is `pydocstyle` choice, not a PEP 257 recommendation? Anyways, using a consistent style is better.

Reply:
Will setup GitHub action with black and blacken-docs.

It the meantime, these concerts have been addressed in the follow up pull request.